TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot_components/robot/SimpleDuckSpinner.java

```diff
@@ -52,6 +54,14 @@ public void stopRunning() {
         servoDirection = 0;
         isRunning = false;
     }
+
+    public boolean isPowered () {
```

Comment:
remove getRuntime and instead have isPowered return a double- how long it has been running if it is running and -1 if not running.